solr/server/solr/configsets/_default/conf/solrconfig.xml

```diff
@@ -1146,6 +1146,11 @@
       <str>yyyy-MM-dd HH:mm[:ss[.SSS]][z</str>
       <str>yyyy-MM-dd HH:mm[:ss[,SSS]][z</str>
       <str>yyyy-MM-dd</str>
+      <str>EEE, dd MMM yyyy HH:mm:ss z</str>
+      <str>EEEE, dd-MMM-yy HH:mm:ss z</str>
+      <str>EEE MMM d HH:mm:ss yyyy</str>
+      <str>EEE MMM d hh:mm:ss z yyyy</str>
+      <str>yyyy-MM-dd hh:mm:ss</str>
```

Comment:
This pattern could be removed if we add optional brackets to an existing pattern.

Reply:
Right, it could be joined with "yyyy-MM-dd" as "yyyy-MM-dd [hh:mm:ss]".
Or should I replace the "hh" bit with "HH"?

Reply:
Close; it'd be this:  `yyyy-MM-dd[ hh:mm:ss]`  (notice the bracket starts immediately after the 'dd'; otherwise we'd be wrongly insisting on a trailing space.  

And yes, always use "HH" and not "hh" unless the pattern contains an AM/PM designator (and these don't).